### PR TITLE
Fixes some build issues on Solaris 11.

### PR DESCRIPTION
**Pull request by xsumbe00**

### CODE REVIEW

**Comment by awakecoding** [comment]
@freerdp-bot test.

File: libfreerdp/core/tcp.c [hunk 1]
```diff
@@ -51,6 +51,8 @@
 #include <sys/select.h>
 #endif
 
+#include <sys/filio.h>
```
Comment (bmiklautz) [inline libfreerdp/core/tcp.c]: This is missing a ifdef for sun since it's at least not available on linux.

**Comment by xsumbe00** [comment]
Added missing ifdef.

**Comment by bmiklautz** [comment]
@freerdp-bot test.

**Comment by bmiklautz** [comment]
@freerdp-bot test

**Comment by bmiklautz** [comment]
+1

**Comment by bmiklautz** [comment]
As said above PR looks good but I'd like to re-test it when the windows test build machine is online again.

**Comment by awakecoding** [comment]
@freerdp-bot test.

**Pull request merged**